Gems/Atom/RHI/DX12/Code/CMakeLists.txt

```diff
@@ -12,10 +12,8 @@ ly_get_list_relative_pal_filename(pal_source_dir ${CMAKE_CURRENT_LIST_DIR}/Sourc
 include(${pal_source_dir}/PAL_${PAL_PLATFORM_NAME_LOWERCASE}.cmake) # PAL_TRAIT_ATOM_RHI_DX12_SUPPORTED
 
 if(PAL_TRAIT_PIX_AVAILABLE)
-    set(USE_PIX_DEFINE "USE_PIX")
     set(PIX_BUILD_DEPENDENCY "3rdParty::pix")
 else()
-    set(USE_PIX_DEFINE "")
     set(PIX_BUILD_DEPENDENCY "")
 endif()
```

Comment:
THis can be removed as the DX12 RHI now always has PAL_TRAIT_PIX_AVAILABLE as undefined or false